CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+
+## [0.1.0] - 2019-08-08
+### Added
+
+- Initial release. In the reorganization between @hanamirb and @dry_rb, we're extracting `Hanami::Logger`
+  (from `hanami-utils` gem) into a standalone `dry-logger` gem.
+  
+  
+[0.1.0]: https://github.com/dry-rb/dry-logger/commit/9f2748fc3c60f81043273e7a114567ef02452160
```

Comment:
there was no release yet, so the changelog should be more or less like this:

```
## [Unreleased]

Initial release. In the reorganization between @hanamirb and @dry_rb, we're extracting `Hanami::Logger` (from `hanami-utils` gem) into a standalone `dry-logger` gem.
```

Reply:
Thanks, added!